index.html

```diff
@@ -27,288 +27,400 @@
+                        </ul>
+                    </div>
+                    <div class="project-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </div>
+                    <ul class="tags">
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="javascript">javascript</li>
+                    </ul>
+                    <div class="project-button">
+                        <span class="project-text">
+                            See Project
+                        </span>
+                    </div>
                 </div>
-            </div>
-        </article>
-
-        <article class="first-project">
-            <img src="./Images/2 multi-post stories.png" alt="" class="work-image">
-            <div class="details-container">
-                <div class="heading-and-highlights">
-                    <h1 class="project-heading">Multi-Post<br>Stories</h1>
-                    <ul class="highlights">
-                        <li class="project-type">CANOPY</li>
-                        <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
-                        <li class="which-end">Back End Dev</li>
-                        <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
-                        <li class="project-year">2015</li>
+            </article>
+    
+            <article class="second-project">
+                <img src="./Images/2 multi-post stories.png" alt="" class="second-project-work-image dcw2">
+                <div class="details-container">
+                    <div class="heading-and-highlights">
+                        <h1 class="project-heading">Multi-Post<br>Stories</h1>
+                        <ul class="highlights">
+                            <li class="project-type">CANOPY</li>
+                            <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
+                            <li class="which-end">Back End Dev</li>
+                            <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
+                            <li class="project-year">2015</li>
+                        </ul>
+                    </div>
+                    <div class="project-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </div>
+                    <ul class="tags">
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="javascript">javascript</li>
                     </ul>
+                    <div class="project-button">
+                        <span class="project-text">
+                            See Project
+                        </span>
+                    </div>
```

Comment:
- [ ] You've really done a great job, for putting your project this far. However, there're few things you need to fulfil before your project meet all the requirements. Expected result for every project card should look exactly like the figma sample. On the desktop at breakpoints 768px, the image and project description are distorted. They don't look like how it is on the figma template. Kindly look into that and fix.

### Your Design (Desktop)
![your-page-desktop](https://user-images.githubusercontent.com/26597432/215192186-570b4f3d-643a-438c-a6d7-caced79aa342.png)

### Figma Design (Desktop)
![temp-page-desktop](https://user-images.githubusercontent.com/26597432/215192321-7b770653-38b8-4c70-9d03-0886fdbc2eec.png)

- [ ] On the mobile, the image on your card does not occupy the whole width of the card at breakpoints `767px` which is opposite of what is expected. Kindly take a look and fix it.

### Your Design (Mobile)
![your-page](https://user-images.githubusercontent.com/26597432/215193150-169f37fc-ee54-4577-abe2-5b0e05b5468d.png)

[your-page-video.webm](https://user-images.githubusercontent.com/26597432/215193709-3842cd56-5d70-4d61-83e2-a4e7d28868f5.webm)


![your-page-mobile](https://user-images.githubusercontent.com/26597432/215192957-2899c171-aec4-4998-a46b-7d8d39fc236b.png)

### Figma Design (Mobile)
![temp-page](https://user-images.githubusercontent.com/26597432/215193284-f0066d43-0693-4e46-8df0-afcc447fe3fb.png)

Reply:
[](https://user-images.githubusercontent.com/26597432/215192957-2899c171-aec4-4998-a46b-7d8d39fc236b.png) You have mentioned this screenshot but you haven't given any details about it. Could you please explain if there is something that needs to be fixed?

Reply:
It is part of the sections you need to work on its responsiveness as regards breakpoints.
Look at your headline text, hamburger, and the section background image. Kindly work on them.

Reply:
I have tried to fix the background image but it is originally only as much wide as it is in the screenshot so it doesn't cover the whole page until the background changes at 786px. Could you please explain how you want it fixed?
And I am unable to spot any issue in the headline text and hamburger. Could you please explain to me what is the problem here?

Reply:
I wonder why you said you did not see what's wrong with the headline text, hamburger, and the background image. If your background image was used properly from your css file, it should cover the whole width of the section and every other elements inside that section must not go outside the section irrespective of any breakpoints. With that screenshot now, the hamburger has gone outside the area covered by your background image which means there's an issue with your styling there. Same goes with the headline text, it also goes outside the background image of the section. Beyond reasonable doubt, I believe you must now understand what's wrong with the section. Take your time to fix it. It is part of learning.